.travis.yml

```diff
@@ -14,6 +14,10 @@ env:
 language: python
 python:
   - "2.7"
+matrix:
+  include:
+    - python: "3.4"
+      env: TEST_SUITE=mypyer
 # command to run tests
 script:
   - ./tools/travis/$TEST_SUITE
```

Comment:
`TEST_SUITE` is supposed to be the name of the script that should be run from tools/travis/. I named my script mypyer because the name 'mypy' is already taken by mypy.